### DISCUSSION TITLE
Docustream

### DISCUSSION THREAD

**Discussion by erginozcan1993** [iptv-org/community#693] [Q&A]
**Channel Name:** Docustream
**Channel ID:** Docustream.nl
**Country:** UK
**Language:** English
**Website:** https://www.spiintl.com
**Stream URL (optional):** xxx
**Notes (optional):** Brand new channel in Samsung TV Plus

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19720
